watertap/unit_models/zero_order/tests/test_anaerobic_digestion_reactive_zo.py

```diff
@@ -250,7 +250,6 @@ def test_costing():
 
     assert m.fs.unit.electricity[0] in m.fs.costing._registered_flows["electricity"]
     assert isinstance(m.fs.costing.total_capital_cost, Var)
-    assert isinstance(m.fs.costing.total_fixed_operating_cost, Var)
```

Comment:
Also not part of this PR, but I think we really should implement test harnesses and drop tests like this. If the variable doesn't exist, there will be an error message that it doesn't exist either on build if the model needs it or in a regression test.

Reply:
I agree -- these kinds of tests create "make work" when we refactor the code. In any case the existence of these `Var`s should be tested by the costing package.